File: BackEnd/src/main/java/com/BackEnd/dto/PaymentResponse.java
package com.BackEnd.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class PaymentResponse {
    String code;
    String desc;
    PayosData data;
    String signature;

}
class PayosData{
    String bin;
    String accountNumber;
    String acountName;
    String currency;
    String paymentLinkId;
    int amount;
    String description;
    int orderCode;
    String status;
    String checkoutUrl;
    String qrCode;

}

File: FrontEnd/app/src/main/java/com/example/frontend/data/dto/PaymentResponse.kt
package com.example.frontend.data.dto

data class PayosResponse(
    val code: String,
    val desc: String,
    val payosData: PayosData,
)
data class PayosData(
    val bin: String,
    val accountNumber: String,
    val accountName: String,
    val currency: String,
    val paymentLinkId: String,
    val amount: Int,
    val description: String,
    val orderCode: Int,
    val status: String,
    val checkoutUrl: String,
    val qrCode: String,
)